### PR TITLE
Graphiql

### PR DESCRIPTION
**Pull request by LuchoLeonel**

### CODE REVIEW

File: integration_tests/tests/web_app_certos.rs [hunk 1]
```diff
@@ -465,6 +465,35 @@ mod website {
       check_public_certificate(&d, &title, &format!("Invitation {raw_description}"), &image).await;
     }
 
+
+    integration_test!{ use_graphiql (c, d)
+      async fn send_graphql_query(d : &Selenium, query: &str) {
+        d.goto("http://localhost:8000/#").await;
+        d.click("#graphiql").await;
+        d.driver.maximize_window().await.expect("to maximize window");
```
Comment (nubis) [inline integration_tests/tests/web_app_certos.rs:473]: This "maximize_window" should be added to all tests then.

File: web_app_certos/src/views/graphiql.tsx [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+      setHeaders(JSON.stringify({Authentication: await getRawAuthorization(graphqlUrl, "POST", maybeNewBody)}));
+      setBody(maybeNewBody);
+      setHeadersSetted(true);
+    }
+  }
+
+  if (!schema) return <Loading />;
+
+  return(
+    <Box sx={{height: "100vh"}} id={headersSetted ? "header-setted" : "header-not-setted"}>
```
Comment (nubis) [inline web_app_certos/src/views/graphiql.tsx:49]:
```suggestion
    <Box sx={{height: "100vh"}} id={ headers ? "header-set" : "header-not-set"}>
```

File: web_app_certos/src/views/graphiql.tsx [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+import { envs } from "../components/cypher";
+import Loading from "./loading";
+import 'graphiql/graphiql.css';
+
+
+const Graphiql = () => {
+  const [schema, setSchema] = useSafeSetState<GraphQLSchema>();
+  const [body, setBody] = useSafeSetState<string | boolean>();
+  const [headers, setHeaders] = useSafeSetState<string>("");
+  const [headersSetted, setHeadersSetted] = useSafeSetState<boolean>(false);
```
Comment (nubis) [inline web_app_certos/src/views/graphiql.tsx:18]: Let's not use headersSetted, just use headers and set it to an empty string when not set.

File: integration_tests/tests/web_app_certos.rs [hunk 4]
```diff
@@ -465,6 +465,35 @@ mod website {
...
+        d.click(".graphiql-tab-add").await;
+        d.click(".graphiql-query-editor").await;
+        d.fill_in(".graphiql-query-editor textarea", query).await;
+        d.wait_for("#header-setted").await;
```
Comment (nubis) [inline integration_tests/tests/web_app_certos.rs:477]: Esto cambia.

File: integration_tests/tests/web_app_certos.rs (hunk 1 above)
Comment (gerar10) [reply integration_tests/tests/web_app_certos.rs:473]: the following line is added in the function "start" implemented for the struct Selenium inside the selenium.rs file: 

`driver.maximize_window().await.expect("to maximize window");`

File: web_app_certos/src/views/graphiql.tsx (hunk 3 above)
Comment (gerar10) [reply web_app_certos/src/views/graphiql.tsx:18]: Done

File: integration_tests/tests/web_app_certos.rs (hunk 4 above)
Comment (gerar10) [reply integration_tests/tests/web_app_certos.rs:477]: the `d.wait_for("#header-setted").await;` is modified by the following: 

`d.wait_for("#header-set").await;`

**Review by nubis — APPROVED** [review]

**Pull request merged**